urdf/crane_x7_wide_two_finger_gripper.xacro

```diff
@@ -78,6 +78,11 @@
       </inertial>
     </link>
 
+    <joint name="${name_joint_gripper_finger_b}_mimic" type="fixed">
```

Comment:
Maybe we should add a comment with a link to the issue where they explain this fix?